chatkit-core/src/test/kotlin/com/pusher/chatkit/rooms/state/RoomUpdatedTest.kt

```diff
@@ -0,0 +1,29 @@
+package com.pusher.chatkit.rooms.state
+
+import assertk.assertThat
+import assertk.assertions.isNotNull
+import com.pusher.chatkit.state.ChatkitState
+import com.pusher.chatkit.state.RoomUpdated
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+
+class RoomUpdatedTest : Spek({
+
+    describe("given a joined room state with one room") {
+        val givenState = ChatkitState(
+                joinedRoomsState = JoinedRoomsState(
+                        mapOf(JoinedRoomsStateTestUtil.roomOneId to JoinedRoomsStateTestUtil.roomOne),
+                        mapOf(JoinedRoomsStateTestUtil.roomOneId to 1)))
+
+        describe("when an event for updating a room that is part of the state is received") {
+            val newState = roomUpdatedReducer(givenState,
+                    RoomUpdated(JoinedRoomsStateTestUtil.roomOneUpdated))
+
+            it("then the state should contain the updated room") {
+                assertThat(newState.joinedRoomsState).isNotNull()
+                        .containsOnly(JoinedRoomsStateTestUtil.roomOneId
+                                to JoinedRoomsStateTestUtil.roomOneUpdated)
```

Comment:
indeed [removing the util objec](https://github.com/pusher/chatkit-android/pull/192/files#r387113504)t will allow us to have more legible tests and not break lines within pair definition

Reply:
👍